lisa/tools/swap.py

```diff
@@ -1,6 +1,7 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
 
+from math import e
```

Comment:
What's it? Is it misimported?

Reply:
it's brought by rebase I think. removed now.